.husky/.gitignore

```diff
@@ -0,0 +1 @@
+_
```

Comment:
@suzubara heads up, in order to commit after the husky version update in https://github.com/trussworks/react-uswds/pull/867/commits/1d091e847a28b36c2da38f13d5ea92e6542be880 I just had to install husky which added this file.

Reply:
Actually, I'm not so sure I did everything required here. In https://github.com/trussworks/react-uswds/pull/877 the patch notes mentions updating from husky 4 to 5 being more involved:

>### Migrating
>Important Husky v5 brings a lot of improvements but is also very different from v4. Git hooks won't work if you only upgrade husky dependency, existing config needs to be migrated too.

>The best way to switch to v5 is to follow the new installation instructions and migrate existing hooks command using husky add.

I found https://github.com/typicode/husky-4-to-5 via https://github.com/typicode/husky/issues/854#issuecomment-776126582

Reply:
@suzubara @ahobson Okay, so the husky 5 docs don't cover the implementation of how husky 5 works very well (they no longer use package.json, and instead rely on scripts in the .husky directory). Everything is updated in accordance with the [Husky 5 docs](https://typicode.github.io/husky/#/?id=install) now via 58b6d22